content/sensu-go/6.4/operations/maintain-sensu/tune.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Tune Sensu"
+linkTitle: "Tune Sensu"
+description: "Learn when and how to tune Sensu to resolve performance issues, with links to more detailed documentation."
+weight: 25
+version: "6.4"
+product: "Sensu Go"
+menu:
+  sensu-go-6.4:
+    parent: maintain-sensu
+---
+
+If you consistently experience performance issues with your Sensu installation, these tuning options can help restore proper operation.
+
+
+{{% notice note %}}
+**NOTE** Read [Troubleshoot Sensu](../troubleshoot/) before you tune your Sensu installation.
+The troubleshooting page describes common problems and may resolve your issue without tuning adjustments.<br><br>
+The [Hardware requirements](../../deploy-sensu/hardware-requirements/) and [Deployment architecture for Sensu](../../deploy-sensu/deployment-architecture/) pages also include helpful information about planning and optimizing your Sensu installation; configuration, networking, and cloud recommendations; and architecture-specific considerations.
+{{% /notice %}}
+
+## Latency tolerances for etcd
+
+If you use embedded etcd for storage, you might notice high network or storage latency.
```

Comment:
We should be more detailed about what customers will actually notice that indicates high network or storage latency.